Core/HLE/__sceAudio.cpp

```diff
@@ -364,6 +380,15 @@ void __AudioUpdate() {
 
 	if (g_Config.bEnableSound) {
 		resampler.PushSamples(mixBuffer, hwBlockSize);
+		if (m_logAudio)
+		{
+			s16 *clamped_data = new s16[hwBlockSize * 2];
```

Comment:
It looks like this memory is never freed.  Should we use a `vector` instead or maybe a member/static global so we don't need to reallocate every frame?

-[Unknown]